examples/platforms/nrf528xx/src/radio.c

```diff
@@ -970,3 +1012,59 @@ uint32_t nrf_802154_random_get(void)
 {
     return otRandomNonCryptoGetUint32();
 }
+
+uint64_t otPlatRadioGetNow(void)
+{
+    return otPlatTimeGet();
+}
+
+#if OPENTHREAD_CONFIG_CSL_RECEIVER_ENABLE
```

Comment:
I changed it back to `OPENTHREAD_CONFIG_CSL_RECEIVER_ENABLE`... The reason is that there's a macro called `TIME_SYNC` which needs `OPENTHREAD_CONFIG_MAC_HEADER_IE_SUPPORT` and it is unrelated to `updateIeData`. So if we compile as: `TIME_SYNC=1 make -f examples/Makefile-simulation`, it would cause an error that `updateIeData` is defined but not used.